Update geo-steering.md (#9692)
cloudflare/cloudflare-docs: content/load-balancing/understand-basics/traffic-steering/steering-policies/geo-steering.md

Any data center not explicitly defined will fall back to using the corresponding `country_pool`, then `region_pool` mapping (if it exists), and finally to associated default pools.

{{<Aside type="note">}}PoP steering is only available to Enterprise customers and only accessible via the API.{{</Aside>}}

### Failover behavior

> A fallback pool will be used if there is only one pool in the region and it is unavailable.

Reviewer: ```suggestion
A fallback pool will be used if there is only one pool in the same region and it is unavailable.
```

> When there is one pool in the region and it is unavailable, if there is a fallback pool, it would be used.

Reviewer: ```suggestion
A fallback pool will be used if there is only one pool in the region and it is unavailable.
```

> If there are some pools in the region, the order of the pool will be respected. For example, when the 1st pool is down, 2nd pool will be used.

Reviewer: ```suggestion
If there are multiple pools in the same region, the order of the pools will be respected. For example, if the first pool is unavailable, the second pool will be used.
```
